scripts/dump_meta_data.sql

```diff
@@ -0,0 +1,109 @@
+-- This script will dump relevant meta data from internal TimescaleDB tables
+-- that can help our engineers trouble shoot.
+--
+-- usage:
+-- psql [your connect flags] -d your_timescale_db < dump_meta_data.sql > dumpfile.txt
+
+-- List extensions
+\dx
+
+-- Get potential build tag
+SELECT * FROM _timescaledb_internal.get_git_commit();
+
+-- List tables
+\dt
+
+-- List hypertables
+SELECT * FROM _timescaledb_catalog.hypertable;
+
+-- List hypertable indexes
+SELECT * FROM _timescaledb_catalog.hypertable_index;
+
+-- List chunk indexes
+SELECT * FROM _timescaledb_catalog.chunk_index;
+
+-- Get size of hypertables
+SELECT hypertable,
+       table_bytes,
+       index_bytes,
+       toast_bytes,
+       total_bytes
+       FROM (
+       SELECT *, total_bytes-index_bytes-COALESCE(toast_bytes,0) AS table_bytes FROM (
+              SELECT
+              h.schema_name || '.' || h.table_name as hypertable,
+              sum(pg_total_relation_size('"' || c.schema_name || '"."' || c.table_name || '"'))::bigint as total_bytes,
+              sum(pg_indexes_size('"' || c.schema_name || '"."' || c.table_name || '"'))::bigint AS index_bytes,
+              sum(pg_total_relation_size(reltoastrelid))::bigint AS toast_bytes
+              FROM
+              _timescaledb_catalog.hypertable h,
+              _timescaledb_catalog.chunk c,
+              pg_class pgc,
+              pg_namespace pns
+              WHERE c.hypertable_id = h.id
+              AND pgc.relname = h.table_name
+              AND pns.oid = pgc.relnamespace
+              AND pns.nspname = h.schema_name
+              AND relkind = 'r'
+              GROUP BY h.schema_name || '.' || h.table_name
```

Comment:
If you do the thing suggested above you can also group by `pgc.oid` here and do the corresponding thing in the projection (column selection).

Reply:
Here I actually want the full name later.

Reply:
You can easily get the name from an OID by casting, e.g., `pgc::regclass::text`

Reply:
But I want the schema as well. Won't I have to join against pg_namespace or something then ? If I have to do that, I prefer this as I already have both the schema and the table name.

Reply:
You will get the fully qualified name.

Reply:
OK, changed.